gym/lib/gym/runner.rb

```diff
@@ -28,6 +28,8 @@ def run
 
         path
       elsif Gym.project.mac?
+        package_app
```

Comment:
I don't think that we want to do this here for every app. There are times where this is not going to be desired behavior. Instead, I think there should be some argument passed to _gym_ to make this happen.